### PR TITLE
Email layout, image and updated style and copy

### PR DESCRIPTION
**Pull request by hammerfunk**
#283 Updating email styles

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@
   config.assets.raise_runtime_errors = true
 
   config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
+  config.action_mailer.asset_host = "http://localhost:3000/"
```
Comment (dreamfall) [inline config/environments/development.rb]: You need to specify host for production env as well.

File: config/environments/development.rb [hunk 2]
```diff
@@ -33,9 +33,13 @@
...
   config.action_mailer.delivery_method = :letter_opener
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
   config.active_job.queue_adapter = :inline
+
+  # configure the devise email layout
+  config.to_prepare do { Devise::Mailer.layout "mailer" }
```
Comment (matthewford) [inline config/environments/development.rb]: unexpected token tSTRING

File: config/environments/production.rb [hunk 3]
```diff
@@ -84,4 +84,7 @@
   config.active_record.dump_schema_after_migration = false
 
   # AWS SES mail settings are in config/initializers/aws_ses.rb
+
+  # configure the devise email layout
+  config.to_prepare do { Devise::Mailer.layout "mailer" }
```
Comment (matthewford) [inline config/environments/production.rb]: unexpected token tSTRING

File: config/environments/staging.rb [hunk 4]
```diff
@@ -80,4 +80,7 @@
   config.active_record.dump_schema_after_migration = false
 
   # AWS SES mail settings are in config/initializers/aws_ses.rb
+
+  # configure the devise email layout
+  config.to_prepare do { Devise::Mailer.layout "mailer" }
```
Comment (matthewford) [inline config/environments/staging.rb]: unexpected token tSTRING

**Pull request merged**